From 3b0fd7dd26fa5af0b310650b2974d800ace80802 Mon Sep 17 00:00:00 2001
From: Mart Raudsepp <leio@gentoo.org>
Date: Fri, 15 Feb 2019 21:34:34 +0200
Subject: [PATCH 2/2] meson.build: Add 32-bit sparc to seccomp unsupported list

32-bit sparc doesn't support seccomp either.

(cherry picked from commit 44b49e6ca02add9592abac03a8578b9eea1a5088)
---
 meson.build | 1 +
 1 file changed, 1 insertion(+)

diff --git a/meson.build b/meson.build
index 50479f5f0..91915b31a 100644
--- a/meson.build
+++ b/meson.build
@@ -72,6 +72,7 @@ unsupported_cpus = [
   'parisc',
   'parisc64',
   'sh4',
+  'sparc',
   'sparc64',
 ]
 system_supports_seccomp = host_system == 'linux'
-- 
2.17.0

